src/cond.d

```diff
@@ -25,6 +25,20 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
+private __gshared Identifier idUnitTest;
```

Comment:
Getting an `Identifier` from a `const char*` requires `strlen` call. These caches can reduce the `version(ident)` lookup cost a little.